charts/helmet/values.yaml

```diff
@@ -402,10 +402,14 @@ exports:
     ## @param envVars Environment variables to be set on APP container
     ## Example:
     ## envVars:
-    ##   - name: FOO
+    ##   FOO: "foo"
+    ##   BAR:
     ##     value: "bar"
+    ##   BAZ:
+    ##     valueFrom:
+    ##       configMapRef: ...
     ##
-    envVars: []
+    envVars: null
```

Comment:
This change was needed to silence a warning about overwriting the default `[]` with a table (aka map aka dictionary) if the consuming chart is providing a map instead.